src/test/regress/expected/multi_modifications.out

```diff
@@ -107,9 +107,8 @@ INSERT INTO limit_orders VALUES (32744, 'AAPL', 9580, '2004-10-19 10:23:54', 'bu
 -- try a single-row INSERT with no shard to receive it
 INSERT INTO insufficient_shards VALUES (32743, 'AAPL', 9580, '2004-10-19 10:23:54', 'buy',
 										20.69);
-ERROR:  distributed modifications must target exactly one shard
-DETAIL:  This command modifies no shards.
-HINT:  Make sure the value for partition column "id" falls into a single shard.
+ERROR:  cannot find shard interval
```

Comment:
For the readers of this PR, this is an error message anyone hits only when there is something wrong in the metadata or the hash function (or such). In other words, this is an error message that is not possible to hit easily.
(Note that even the regression test that triggers this error is done via manual metadata modifications)

Thus, we thought that it is OK to change the error message to be slightly less informative.